tests/test_optics.py

```diff
@@ -25,7 +25,7 @@ def test_sunrise_sunset():
     assert len(sunrise) == len(lat)
 
     # sunrise will be earlier for the SH in January
-    assert sunrise[0] < sunrise[1]
+    assert sunrise[0][0] < sunrise[1][0]
```

Comment:
Does this change the output structure? That worries me a bit. I think it might be good to start with a test that confirms the behavior of the astral version, and then migrate the skyfield implementation, which hopefully assures that the output has the same shape.